### ISSUE TITLE
Waymo self-driving challenge: YOLOR(Scaled-YOLOv4-based) is the best speed/accuracy, better than YOLOv5/Detr

### ISSUE THREAD

**Issue by AlexeyAB** [pjreddie/darknet#2454]
Waymo autonomous driving challenge (Waymo Open Dataset Challenge): Real-time 2D Detection
**YOLOR (Scaled-YOLOv4-based)** is the best in terms of speed/accuracy, the fastest, and more accurate than **YOLOv5 and Detr**: https://waymo.com/open/challenges/2021/real-time-2d-prediction/

----

* **DIDI MapVision** https://arxiv.org/abs/2106.08713
* **YOLOR** https://arxiv.org/abs/2105.04206
    * code (Pytorch): https://github.com/WongKinYiu/yolor
* **Scaled-YOLOv4(CVPR21)**: https://openaccess.thecvf.com/content/CVPR2021/html/Wang_Scaled-YOLOv4_Scaling_Cross_Stage_Partial_Network_CVPR_2021_paper.html
    * code (Pytorch): https://github.com/WongKinYiu/ScaledYOLOv4
* **YOLOv4**: https://arxiv.org/abs/2004.10934
    * code (Darknet, Pytorch, TensorFlow, TRT, OpenCV...): https://github.com/AlexeyAB/darknet#yolo-v4-in-other-frameworks

----

| | |
|---|---|
| <img width="446" alt="APL1" src="https://user-images.githubusercontent.com/4096485/122941628-3c8ac700-d37e-11eb-8919-f6ce65f61ea0.png"> | <img width="450" alt="APL2" src="https://user-images.githubusercontent.com/4096485/122941646-414f7b00-d37e-11eb-8dd3-df55f690c752.png"> |
| https://waymo.com/open/challenges/2021/real-time-2d-prediction/ | https://waymo.com/open/challenges/2021/real-time-2d-prediction/ |
| ![E4dmtbBWEAQuZFN](https://user-images.githubusercontent.com/4096485/122942094-9db29a80-d37e-11eb-8930-89a7c466e7f5.jpg) | ![121721568-126b1680-caed-11eb-81bf-792bb398708a](https://user-images.githubusercontent.com/4096485/122942393-db172800-d37e-11eb-958c-b84208823e45.jpg) |
